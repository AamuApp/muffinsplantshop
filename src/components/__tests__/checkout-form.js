import React from "react";
import { shallow } from 'enzyme';

import CheckoutForm from '../checkout-form/checkout-form';

test('CheckoutForm renders properly', () => {
    const wrapper = shallow(<CheckoutForm />);
    expect(wrapper).toMatchSnapshot();
});