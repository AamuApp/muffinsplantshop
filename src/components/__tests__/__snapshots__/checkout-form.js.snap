// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutForm renders properly 1`] = `
<div>
  <div
    className="checkoutForm__section current"
  >
    <h2>
      1. Shipping
    </h2>
  </div>
  <ShippingForm
    setSection={[Function]}
    setValues={[Function]}
  />
  <div
    className="checkoutForm__section "
  >
    <h2>
      2. Billing
    </h2>
  </div>
  <div
    className="checkoutForm__section "
  >
    <h2>
      3. Payment
    </h2>
  </div>
</div>
`;
